Library/Homebrew/language/node.rb

```diff
@@ -4,6 +4,17 @@ def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache\n"
     end
 
+    def self.pack_for_installation
+      # npm 5.0.0 broke the assumption Homebrew made that npm would always
+      # create and install real directories when passed a file path. We can
+      # restore the old behaviour by packing the path first with "npm pack"
+      # which also helpfully spits out the filename it just created allowing
+      # us to pass that as the final argument.
+      Utils.popen_read("npm pack").chomp
+      return if $?.exitstatus.zero?
+      raise "npm failed to pack #{Dir.pwd}"
+    end
```

Comment:
Or I could just turn the `Utils.popen_read` into a block. I might try that. That feels sane. I've been thinking about this too long today & now my mind is sludge 😆